state/redis/redis.go

```diff
@@ -127,6 +127,10 @@ func parseRedisMetadata(meta state.Metadata) (metadata, error) {
 	return m, nil
 }
 
+func (r *StateStore) Ping() error {
+	return errors.New("not implemented")
```

Comment:
Default implementation should return nil. I think it would make it easier to adopt this feature. Otherwise, daprd would report unhealthy by default.

Reply:
@artursouza Should this be a `Feature` ? If the Feature is present call Ping()? Instead of default nil implementation.

Reply:
I think returning nil by default is fine. We can make this a feature once there is specific behavior in a method that can impact decisions above. I don't think this is the case (yet).

Reply:
I have changed the default implementation of `Ping` operation to return nil.